zeebe/broker-client/src/main/java/io/camunda/zeebe/broker/client/impl/BrokerTopologyManagerImpl.java

```diff
@@ -212,10 +212,12 @@ public void onTopologyUpdated(final ClusterTopology clusterTopology) {
     }
     this.clusterTopology = clusterTopology;
     LOG.debug("Received new cluster topology with clusterSize {}", clusterTopology.clusterSize());
```

Comment:
:upside_down_face: This log message is a both noisy _and_ not very informative. I'd suggest to check if it actually changed and only then log the entire topology or the actual change in cluster size, partition count and replication factor.

Reply:
This should be already the case. This method is invoked only if local `ClusterTopology` has changed.

Reply:
Ah okay :+1: 
During scaling we would still see repeated logs like "new cluster size 6" as operations get added and completed.

Reply:
I changed it to log only if clusterSize or replicationFactor is changed. Instead, added a log in GatewayClusterTopologyService to log full topology if it has changed.